docs/CHANGELOG.md

```diff
@@ -421,75 +421,255 @@ Beta release.
-[E0104]: https://quick-lint-js.com/errors/E0104/
-[E0106]: https://quick-lint-js.com/errors/E0106/
-[E0108]: https://quick-lint-js.com/errors/E0108/
-[E0110]: https://quick-lint-js.com/errors/E0110/
-[E0111]: https://quick-lint-js.com/errors/E0111/
-[E0119]: https://quick-lint-js.com/errors/E0119/
-[E0144]: https://quick-lint-js.com/errors/E0144/
-[E0151]: https://quick-lint-js.com/errors/E0151/
-[E0173]: https://quick-lint-js.com/errors/E0173/
-[E0176]: https://quick-lint-js.com/errors/E0176/
-[E0178]: https://quick-lint-js.com/errors/E0178/
-[E0179]: https://quick-lint-js.com/errors/E0179/
-[E0180]: https://quick-lint-js.com/errors/E0180/
-[E0181]: https://quick-lint-js.com/errors/E0181/
-[E0182]: https://quick-lint-js.com/errors/E0182/
-[E0183]: https://quick-lint-js.com/errors/E0183/
-[E0184]: https://quick-lint-js.com/errors/E0184/
-[E0185]: https://quick-lint-js.com/errors/E0185/
-[E0186]: https://quick-lint-js.com/errors/E0186/
-[E0187]: https://quick-lint-js.com/errors/E0187/
-[E0188]: https://quick-lint-js.com/errors/E0188/
-[E0189]: https://quick-lint-js.com/errors/E0189/
-[E0190]: https://quick-lint-js.com/errors/E0190/
-[E0191]: https://quick-lint-js.com/errors/E0191/
-[E0192]: https://quick-lint-js.com/errors/E0192/
-[E0193]: https://quick-lint-js.com/errors/E0193/
-[E0194]: https://quick-lint-js.com/errors/E0194/
-[E0195]: https://quick-lint-js.com/errors/E0195/
-[E0196]: https://quick-lint-js.com/errors/E0196/
-[E0197]: https://quick-lint-js.com/errors/E0197/
-[E0198]: https://quick-lint-js.com/errors/E0198/
-[E0199]: https://quick-lint-js.com/errors/E0199/
-[E0203]: https://quick-lint-js.com/errors/E0203/
-[E0205]: https://quick-lint-js.com/errors/E0205/
-[E0207]: https://quick-lint-js.com/errors/E0207/
+## Contributors
+
+- [AidenThing](https://github.com/AidenThing)<br>
+- [Amir](https://github.com/ahmafi)<br>
+- [Christian Mund](https://github.com/kkkrist)<br>
+- [Daniel La Rocque](https://github.com/dlarocque)<br>
+- [Dave Churchill](https://www.cs.mun.ca/~dchurchill/)<br>
+- [David Vasileff](https://github.com/dav000)<br>
+- [Erlliam Mejia](https://github.com/erlliam)<br>
+- [Himanshu](https://github.com/singalhimanshu)<br>
+- [Jenny "Jennipuff" Wheat](https://twitter.com/jennipaff)<br>
+- [Jimmy Qiu](https://github.com/lifeinData)<br>
+- [Kim "Linden"](https://github.com/Lindenbyte)<br>
+- [Lee Wannacott](https://github.com/LeeWannacott)<br>
+- [Matheus de Sousa](https://github.com/keyehzy)<br>
+- [Nico Sonack](https://github.com/herrhotzenplotz)<br>
+- [Piotr Dąbrowski](https://github.com/yhnavein)<br>
+- [Shivam Mehta](https://github.com/maniac-en)<br>
+- [coc.nvim](https://github.com/neoclide/coc.nvim)<br>
+- [config-global-groups](https://quick-lint-js.com/config/#global-groups)<br>
+- [tiagovla](https://github.com/tiagovla)<br>
+- [wagner riffel](https://github.com/wgrr)<br>
+
+## Error codes
+
+- [E0001](https://quick-lint-js.com/errors/E0001/): variable assigned before its declaration<br>
```

Comment:
Must fix: A list of error codes doesn't belong in CHANGELOG.md. I don't know what the purpose of this change is for. Undo.